scripts/ci/run-ci-tests.sh

```diff
@@ -237,7 +218,7 @@ bash -x ./test/jenkins/criu-inhfd.sh
 if [ -z "$SKIP_EXT_DEV_TEST" ]; then
 	make -C test/others/mnt-ext-dev/ run
 fi
-#make -C test/others/exec/ run
```

Comment:
Heh it looks like `make -C test/others/exec/ run` is broken long ago...
```
+ ../../../criu/criu exec -t 186212 fake_syscall
...
Error (criu/crtools.c:162): excessive parameter for command exec
```
As criu exec is already deprecated.

Reply:
Good point. Maybe we should just remove that directory.

Reply:
Let me open a PR to remove that directory.